bloom-housing/bloom
# feat: lottery endpoints

This PR addresses #4055 


## Description
This creates the lottery results generation endpoint and the lottery export as csv

another ticket will cover converting that export to a spreadsheet and include the preference sheets

## How Can This Be Tested/Reviewed?
- create applications on a listing (or use a listing with existing applications)
- navigate to the listings' lottery tab
- you should be able to click the run lottery button and lottery results should get generated
- you should be able to click the export results button and get a csv spreadsheet with the lottery results

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
